package.json

```diff
@@ -36,34 +41,25 @@
   },
   "homepage": "https://github.com/aragon/aragen#readme",
   "dependencies": {
+    "@aragon/cli-utils": "^0.0.3",
     "@babel/polyfill": "^7.0.0",
     "chalk": "^2.1.0",
-    "figures": "^3.0.0",
-    "ganache-core": "2.5.3",
+    "ganache-core": "^2.5.7",
     "listr": "^0.14.3",
     "ncp": "^2.0.0",
     "rimraf": "^2.6.2",
-    "web3": "1.0.0-beta.35",
-    "yargs": "^12.0.0"
+    "web3": "^1.2.0",
+    "yargs": "^13.2.4"
   },
   "devDependencies": {
-    "@aragon/cli": "^5.0.0",
+    "@aragon/cli": "^6.0.0",
     "@aragon/id": "^2.0.0",
     "@aragon/os": "^4.2.0",
     "eth-ens-namehash": "^2.0.8",
-    "ganache-cli": "~6.2.0",
+    "ganache-cli": "^6.4.2",
```

Comment:
@0xGabi It looks like this is actually breaking some assumptions we hold with ETH nodes :(.

The newer versions of ganache-cli do not seem to react properly to an `eth_estimateGas`, making transaction pathing in the client fail (it currently always says you can directly do every action).